src/gui/mainwindow.cpp

```diff
@@ -51,6 +64,7 @@ MainWindow::MainWindow(QWidget* parent) : QMainWindow(parent), ui(new Ui::MainWi
 }
 
 MainWindow::~MainWindow() {
+    delete comm;
     delete dConfig;
     delete dDebug;
     delete dAbout;
```

Comment:
Not needed, since they're children of `MainWindow` and so are deleted automatically. More info: [Object Trees & Ownership](https://doc.qt.io/qt-5/objecttrees.html).

Reply:
Removed